src/assets/stylesheets/index.scss

```diff
@@ -1367,6 +1367,13 @@ section:focus >,
   font-family: var(--jr-headline-font-family), sans-serif;
   font-weight: var(--jr-headline-font-weight);
 }
+.dropdown-item {
+  color: var(--bs-dropdown-link-color) !important;
+}
```

Comment:
If the surrounding background color is the "Primary color" it looks like this:

<img width="1002" alt="Screenshot 2025-03-20 at 10 27 13" src="https://github.com/user-attachments/assets/fdedca99-2d4d-48b9-9269-e276010ddc83" />

Can we adjust the background color automatically? E.g. to `bg-secondary`? A mockup of this looks like this:


<img width="1002" alt="Screenshot 2025-03-20 at 10 31 42" src="https://github.com/user-attachments/assets/5cd4858c-287b-459f-8c79-149a6d9052e1" />